src/test/ui/bad/destructuring-assignment.rs

```diff
@@ -0,0 +1,21 @@
+struct S { x: u8, y: u8 }
```

Comment:
I've made a new folder and renamed the test. `src/test/ui/type` didn't seem appropriate consider the existing tests there.

Reply:
Hmm... a folder just for the one test doesn't seem worth it -- It would be if more tests would be added in the future, which is what I assume you're going for... but right now I don't know that I would be in favor of the feature itself... but I guess this is better than `bad/`. ^^